controller/service.go

```diff
@@ -12,24 +12,40 @@ import (
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
+const (
+	birthdayDatabase        = "birthdayDB"
+	birthdayCollection      = "birthdays"
+	personalNumberParameter = "personalNumber"
+	dateParameter           = "date"
+	nameParameter           = "name"
+)
+
+// Service is a type of every method
 type Service struct {
 	Database           *mongo.Database
 	BirthdayCollection *mongo.Collection
+	pb.UnimplementedBirthdayFunctionsServer
 }
 
-type BirthdayStruct struct {
+// BirthdayModel is a model for coverting an object for type: birthdayObject
+type BirthdayModel struct {
 	Name           string
 	Date           string
 	PersonalNumber string
 	ID             string `bson:"_id" json:"id,omitempty"`
 }
 
+// NewService is used each time i create a method
 func NewService(url string) *Service {
 	s := &Service{}
+	// s.BirthdayCollection = mongoConnect.CNX.Database(birthdayDatabase).Collection(birthdayCollection)
+	s.BirthdayCollection = mongoConnect.CNX.Database("birthdayDB").Collection("birthdays")
+
 	return s
 }
 
-func (bs *BirthdayStruct) asBirthdayObject() *pb.BirthdayObject {
+// asBirthdayObject is making any object type to: BirthdayObject type
```

Comment:
```suggestion
// asBirthdayObject converts the BirthdayModel type to: BirthdayObject type
```
It doesn't convert any object, but only `BirthdayModel` type

Reply:
done